### PR TITLE
Bump HTTP poller version to auto-retry

### PR DESCRIPTION
**Pull request by andrewvc**
This will help in the event that a remote server has broken keep-alive as in #39

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,2 +1,4 @@
 source 'https://rubygems.org'
-gemspec
\ No newline at end of file
+gemspec
+#gem "logstash-core", :path => "/Users/andrewvc/projects/logstash"
```
Comment (suyograo) [inline Gemfile]: Can we remove these commented out lines?

File: Gemfile [hunk 2]
```diff
@@ -1,2 +1,4 @@
...
+#gem "logstash-devutils", :path => "/Users/andrewvc/projects/logstash-devutils"
```
Comment (suyograo) [inline Gemfile]: remove commented out lines

**Comment by suyograo** [comment]
@andrewvc remove commented out lines in Gemfile and good to merge

**Comment by andrewvc** [comment]
@suyograo removed the commented out lines, will merge.

**Pull request closed without merging**